nibabel/filebasedimages.py

```diff
@@ -212,16 +211,16 @@ def __getitem__(self):
         '''
         raise TypeError("Cannot slice image objects.")
 
+    @deprecate_with_version('get_header method is deprecated.\n'
+                            'Please use the ``img.header`` property '
+                            'instead.',
+                            '2.1', '4.0')
```

Comment:
This was at least deprecated in the docs at [2.0.0](https://github.com/nipy/nibabel/blob/2.0.0/nibabel/spatialimages.py#L25). Is this meant to indicate the first release a warning was raised?